frontend/src/components/common/form/RadioGroup.tsx

```diff
@@ -72,6 +75,7 @@ export const RadioGroup = ({
     <StyledRadioGroup
       controlId={`input-${field}`}
       className={classNames(!!required ? 'required' : '', className)}
+      $isFlexColumn={flexDirection ? flexDirection === 'column' : true}
```

Comment:
You really don't need this intermediate "$isFlexColumn" here. I think it could be dropped in favor of `flexDirection` (which you exposed in the component props..)

```tsx

export const RadioGroup = ({
  flexDirection = 'column' // so it sets it to column if not supplied
  ...rest
}) => {
 // ...
 <StyledRadioGroup
   flexDirection={flexDirection}
/>
}

export const StyledRadioGroup = styled(Form.Group)`
  &.form-group {
    display: flex;
    margin-bottom: 0;
    .radio-group {
      display: flex;
      flex-direction: column;
      row-gap: 0.4rem;
      flex-direction: ${props => props.flexDirection};  // we already assigned a default above
      row-gap: ${props => (props.flexDirection == 'column' ? '0.4rem' : 'unset')};
      column-gap: ${props => (props.flexDirection == 'column' ? 'unset' : '0.4rem')};
`;

Reply:
Yeah, I was a tad torn on this. I changed to compare against the string column.